phpfarm/docker.sh

```diff
@@ -17,14 +17,23 @@ do
 
     # compile the PHP version
     ./compile.sh $VERSION
+
+    # PHP 5.1 names both CLI and CGI executables 'php'. Rename and move them to
+    # correct dirs. See http://php.net/manual/bg/features.commandline.php
+    if [ "$V" == "5.1" ]; then
+        post='custom/post-install-5.1.6.sh'
+        echo "Running commands from '$post'"
+        /bin/bash "$post" "$VERSION"
+    fi
```

Comment:
I see. I wonder if this could be fixed by another patch before compiling.

My best guess would be changing `SAPI_CGI_PATH=sapi/cgi/php` to `SAPI_CGI_PATH=sapi/cgi/php-cgi` in `sapi/cgi/config9.m4`